### ISSUE TITLE
DVPN keeping stopping after upgrade hh machine to v0.5.2

### ISSUE THREAD

**Issue by alyasboytigas** [HandyOSS/HandyHost#56]
Since I upgrade my HandyHost machine to v0.5.2 I notice sentinel (DVPN) been acting up lately keeps stopping so many times. 

![Screenshot (148)](https://user-images.githubusercontent.com/57547739/152197218-fce9fcb2-66c3-4f58-9313-a450bac48543.png)

**Comment by Femkecl** [comment]
Issue is already adressed here : https://github.com/HandyOSS/HandyHost/issues/49

**Comment by alexsmith540** [comment]
Yeah a me and our testing team ran into this the other day as well, seemed like DVPN's RPC server was having an issue which is why we see those unmarshalling issues from the DVPN RPC service. It seemed to be fixed as of yesterday when I last checked.
